pkg/resource/logging-credentials/logging_operator_secrets.go

```diff
@@ -100,18 +101,40 @@ func UpdateLoggingCredentials(loggingCredentials *v1.Secret) bool {
 		loggingCredentials.Data["readpassword"] = []byte(password)
 		secretUpdated = true
 	}
-	if _, ok := loggingCredentials.Data["writeuser"]; !ok {
-		loggingCredentials.Data["writeuser"] = []byte("write")
+
+	// Check credentials for [clustername]
+	clusterName := lc.GetClusterName()
+	if _, ok := loggingCredentials.Data[fmt.Sprintf("%suser", clusterName)]; !ok {
+		loggingCredentials.Data[fmt.Sprintf("%suser", clusterName)] = []byte(clusterName)
 		secretUpdated = true
 	}
-	if _, ok := loggingCredentials.Data["writepassword"]; !ok {
+	if _, ok := loggingCredentials.Data[fmt.Sprintf("%spassword", clusterName)]; !ok {
 		password, err := genPassword()
 		if err != nil {
 			return false
 		}
-		loggingCredentials.Data["writepassword"] = []byte(password)
+		loggingCredentials.Data[fmt.Sprintf("%spassword", clusterName)] = []byte(password)
+		secretUpdated = true
+	}
+
+	return secretUpdated
+}
+
+// RemoveLoggingCredentials - Remove credentials from LoggingCredentials secret
+func RemoveLoggingCredentials(lc loggedcluster.Interface, loggingCredentials *v1.Secret) bool {
+	var secretUpdated bool = false
+
+	// Check credentials for [clustername]
+	clusterName := lc.GetClusterName()
+
+	if _, ok := loggingCredentials.Data[fmt.Sprintf("%suser", clusterName)]; ok {
```

Comment:
Maybe extract the 2 sprintf into variables for performance?

Reply:
I don't think there's much of a performance impact because it does not seem to be run often.
However I agree that's quite ugly.

But I was more thinking about changing the way this secret is built, but in a separate/future PR.
Today:
```
data:
  gausspassword: [base64 pass]
  gaussuser: Z2F1c3M=
  readpassword: [base64 pass]
  readuser: cmVhZA==
  sm2t7password: [base64 pass]
  sm2t7user: c20ydDc=
```

I think we'd better have:
```
data:
  gauss: [base64 user and pass structure]
  read: [base64 user and pass structure]
  sm2t7: [base64 user and pass structure]
```

Reply:
That New format would bé better. The reason I was proposing a function is that this specific printf is repeated often and i'm a huge believer of DRY